src/core/tests/pass/serialization/custom_ops.cpp

```diff
@@ -29,25 +29,23 @@ class CustomOpsSerializationTest : public ::testing::Test {
     }
 };
 
-class FrameworkNodeExtension : public InferenceEngine::IExtension {
```

Comment:
IMO, `Extension` should in the class name. It is quite confusing in opposite

Reply:
This is not extension anymore. This is just a custom operation